packages/fxa-settings/package.json

```diff
@@ -70,13 +70,14 @@
       ]
     },
     "transformIgnorePatterns": [
-      "[/\\\\]node_modules[/\\\\](?!@mozilla/glean).+\\.(js|jsx|mjs|cjs|ts|tsx)$",
-      "^.+\\.module\\.(css|sass|scss)$"
+      "[/\\\\]node_modules[/\\\\](?!sinon|@mozilla/glean).+\\.(js|jsx|mjs|cjs|ts|tsx)$",
+      "^.+\\.module\\.(css|sass|scss)$",
+      "/node_modules/(?!(sinon|@mozilla/glean)/)"
```

Comment:
What's the difference between this and line 73?  It looks like this is a superset of 73 since it doesn't specify the file extensions?

Reply:
I think you're right – these changes were made over multiple commits and I didn't take the time to optimize.  I just pushed a combined version.